ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -13,6 +13,10 @@ import com.gu.recipeasy.models._
 import com.gu.recipeasy.views
 import models._
 import models.CuratedRecipeForm._
+import auth._
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
You can use the play execution context: `play.api.libs.concurrent.Execution.Implicits._`. https://www.playframework.com/documentation/2.5.x/ThreadPools